API.md

```diff
@@ -27,6 +27,11 @@ interface VoterMailBallot {
   ballotStyle: string,
   precinctId: string,
   ballotfile: string, // filepath?
+}
+```
+
+```
+interface VoterMailBallotTracking extends VoterMailBallot {
```

Comment:
Why extend it? This could just be separate, right? If I were modeling this in a db, I'd probably have a `print_jobs` table whose rows reference a voter by ID.

Reply:
+1, just a reference is good.

Reply:
I'm happy to do this whatever way y'all suggest here, I really have no super strong opinion. 

@brian - For my educational benefit, would you explain why a separate table for the tracking data would be preferable to one table of voter data? 

I had considered separating this data into a different table, but then couldn't think of when the voter data was going to be requested separate from the tracking data in this app and figured that it'd be better to have all the data in one table.